Add Debian to installation.md (#733)
hyprwm/hyprland-wiki: pages/Getting Started/Installation.md

following the instructions
[here](https://github.com/hyprwm/Hyprland/discussions/284)

{{% /details %}}

> {{% details title="Debain*" closed="true" %}}

Author: Should be resolved

Hyprland recently made it into the SID repos and can be installed with

```sh
sudo apt  install hyprland
```
Note: Soon will be in the trixie repos, but as Hyprland is bleeding edge it is only recommend to install with SID even if any other repo got hyprland.

Alternatively, you can also follow the instructions under
["Manual (Manual Build)"](#manual-manual-build) to build Hyprland yourself.

> Note: _Hyprland is not available for Bookworm as it packages are too old

Reviewer: ```suggestion
{{< callout type=info >}}

Hyprland is not available for Bookworm as its packages are too old.

{{< /callout >}}
```
